tests/pass/record-defs.ncl

```diff
@@ -29,6 +29,10 @@ let Assert = fun l x => x || %blame% l in
 ({foo.bar.baz = 1, bar.baz.foo = foo.bar.baz + 1}
   == {foo = {bar = {baz = 1}}, bar = {baz = {foo = 2}}}
   | #Assert) &&
+({foo.bar.baz = 1, bar.baz.foo = foo.bar.baz + 1}
+  == {foo = {bar = {baz = 1}}, bar = {baz = {foo = 2}}}
+  | #Assert) &&
```

Comment:
This looks identical to the test above it?

Reply:
Right, looks like a conflict resolution glitch or a bad copy/paste. Let me get rid of it